crates/core/Cargo.toml

```diff
@@ -56,12 +56,13 @@ stretto = { features = ["async", "sync"], version = "0.8" }
 tar = { version = "0.4.38" }
 thiserror = "1"
 tokio = { features = ["fs", "macros", "rt-multi-thread", "sync", "process"], version = "1" }
-tokio-tungstenite = "0.20"
+tokio-tungstenite = "0.21.0"
 tower-http = { features = ["fs", "trace"], version = "0.5" }
 ulid = { features = ["serde"], version = "1.1" }
 unsigned-varint = "0.7"
 wasmer = { features = ["sys"], workspace = true }
 xz2 = { version = "0.1" }
+enum-iterator = "1.4.1"
```

Comment:
why do we need this dependency

Reply:
I was planning to use it but forgot - but I'm using it now, it's for iterating through all ResourceType values.